controls/emet.rb

```diff
@@ -0,0 +1,16 @@
+# encoding: utf-8
+
+title 'Ms EMET'
+
+control 'EMET-1' do
+  impact 0.7
+  title 'Ms EMET is running'
+  desc 'EMET process monitoring is active'
+  describe processes('EMET_Service.exe') do
```

Comment:
EMET is an addon. EOL announced for 2018.
still recommended by many.
https://insights.sei.cmu.edu/cert/2016/11/windows-10-cannot-protect-insecure-applications-like-emet-can.html